server.js

```diff
@@ -1,6 +1,7 @@
 import './server-shim.js';
 import { DOMParser } from 'linkedom';
 import { CustomElementRender } from './CustomElementRenderer.js';
+import dsd_polyfill_url from './dsd-polyfill.min.js?url';
```

Comment:
can we use import.meta for this? im not huge on non-standard imports like this

Reply:
If you utilize vite and its 'lib' build mode this solution doesn't work anymore, because the asset will always be transformed to a data-url! :(

This was one of the reason, why I wrote the polyfill-inject-plugin finally.